dev/dags/dbt/jaffle_shop/profiles.yml

```diff
@@ -16,7 +16,7 @@ postgres_profile:
   outputs:
     dev:
       type: postgres
-      dbname: "{{ env_var('POSTGRES_DATABASE') }}"
+      dbname: "{{ env_var('POSTGRES_DB') }}"
```

Comment:
Why did we change this?

Reply:
I pointed out the SQLite test to use this profile, but ENV `POSTGRES_DATABASE` is not set in GitHub action, so I just renamed it the env name is already available in CI. But do let me know if it could have any side effect

Reply:
Thanks for the explanation, @pankajastro , I was confused - but it makes sense!